Automattic/jetpack
# Dev Tools: Add new products to the Dev Tools card

Depends on #20344.

#### Changes proposed in this Pull Request:
* Add new list items/radio buttons in the Dev Tools card for new product slugs that will be introduced as Backup and Security are migrated to be real-time for everyone.
* Move Daily and Real-Time items to the bottom of the plans list under a heading marked "— DEPRECATED —", for easy identification and quick removal later.

#### Jetpack product discussion

`pbtFFM-XN-p2`

#### Does this pull request change what data or activity we track or use?

No ✅ 

#### Testing instructions:

**IMPORTANT:** As mentioned above, #20344 must also be applied for this PR to function 100% correctly; see testing instructions for more details.

* For any Jetpack site, open WP-Admin and navigate to the Dashboard > My Plan page.
* Scroll to the bottom of the page, and click the footer link marked "Dev Tools."
* In the panel that opens in the bottom right of your browser window, verify you see the new Security and Backup items (200 GB and 2 TB), and that the Daily/Real-Time offerings are under a "deprecated" heading.
* Click around on any of the listed plans and observe that the active plan changes as you do so. *(NOTE: Clicking any of the new Security or Backup list items will cause an error/loading state if #20344 isn't also applied.)*

##### Reference screenshots (before / after)

<img width="203" alt="image" src="https://user-images.githubusercontent.com/670067/125509224-527bb6f5-71ba-4487-948f-3d33f9fc3f65.png"> <img width="197" alt="image" src="https://user-images.githubusercontent.com/670067/125509118-b47579e7-2f3f-44f4-895f-089c3f444027.png">
